consensus/state.go

```diff
@@ -2439,3 +2454,20 @@ func repairWalFile(src, dst string) error {
 
 	return nil
 }
+
+func (cs *State) hasPrevotedHash(b cmtbytes.HexBytes) bool {
```

Comment:
Do we ever clear the pre-voted hashes, say when the block number increments?

Reply:
I believe this is done in 'updateToState'